.github/workflows/ci.yml

```
@@ -632,6 +632,13 @@ jobs:
         if: matrix.os != 'windows'
       - run: ${{ matrix.execution.shell-ex }} $PARALLEL modin/numpy/test
       - run: ${{ matrix.execution.shell-ex }} -m "not exclude_in_sanity" modin/pandas/test/test_io.py --verbose
+        if: matrix.execution.name != 'unidist'
+      - uses: nick-fields/retry@v2
+        with:
```

@YarShev  Like this?
```suggestion
      - uses: nick-fields/retry@v2
        # to avoid issues with non-stable `to_csv` tests
        # for details see: https://github.com/modin-project/modin/pull/6776
        with:
```

---

Corrected your suggestion a little and also I think we should put some lines in the PR discription like "`to_csv` tests for Modin on MPI through unidist component hang sporadically so this PR make tests rerun to avoid sporadic failures due to timeout."

---

done